test/integration/exceptionalOpeningTimes.js

```diff
@@ -18,38 +18,55 @@ function mockCurrentDate2() {
 }
 
 function expectExceptionalOpeningTimes1($, rows, times) {
-  expect($(rows[0]).text()).to.include('Monday 29 May');
+  expect($(rows[0]).text()).to.include('Thursday 25 May');
   expect($(rows[0]).text()).to.include(times[0]);
+  expect($(rows[1]).text()).to.include('Monday 29 May');
+  expect($(rows[1]).text()).to.include(times[1]);
 }
 
 function expectExceptionalOpeningTimes2($, rows, times) {
+  expect($(rows[0]).text()).to.not.include('Thursday 25 May');
+  expect($(rows[0]).text()).to.not.include('8am to 12pm');
   expect($(rows[0]).text()).to.include('Monday 29 May');
   expect($(rows[0]).text()).to.include(times[0]);
 }
 
 describe('app', () => {
-  describe('opening times', () => {
-    it('should return exceptional opening times for the date', (done) => {
+  describe('opening times with exceptional opening times in the past', () => {
+    before(() => {
       tk.travel(mockCurrentDate1());
+    });
+
+    after(() => {
+      tk.reset();
+    });
+    it('should return exceptional opening times for the date', (done) => {
       chai.request(app)
```

Comment:
It would be good if this was a little bit more verbose, i.e. 'should only return exceptional opening times within the next two weeks'. 

Being more explicit in the related unit tests would be good too as from the tests it is unclear as to what is considered 'in date' without looking at the code.